app/components/containerInspect/containerInspect.html

```diff
@@ -2,17 +2,22 @@
   <rd-header-title title="Container inspect">
   </rd-header-title>
   <rd-header-content>
-    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: containerInfo.Id})">{{ containerInfo.Name|trimcontainername }}</a> &gt; Inspect
+    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: containerInfo.object.Id})">{{ containerInfo.object.Name|trimcontainername }}</a> &gt; Inspect
   </rd-header-content>
 </rd-header>
 
 <div class="row">
   <div class="col-lg-12 col-md-12 col-xs-12">
     <rd-widget>
       <rd-widget-header icon="fa-icon-circle" title="Inspect">
+        <div class="btn-group" role="group" aria-label="...">
+          <button class="btn btn-sm btn-success" ng-click="toggle()" ng-disabled="!TextView"><i class="fa fa-hand-pointer-o space-right" aria-hidden="true"></i>Tree</button>
+          <button class="btn btn-sm btn-success" ng-click="toggle()" ng-disabled="TextView"><i class="fa fa-file-text-o space-right" aria-hidden="true"></i>Text</button>
```

Comment:
Why not use `uib-btn-radio` [like so](https://github.com/portainer/portainer/blob/develop/app/components/volumes/volumes.html#L33-L43) instead?

Reply:
I like the suggestion. Didn't know about `uib-btn-radio`. Thank you.

However, I tried `ng-model="TextView" uib-btn-radio="false"` and it won't. It is the same with the actual implementation if I replace `ng-click="toggle()"` with `ng-click="TextView=!TextView "`. In both cases, buttons behave properly, but the change is not applied to `rd-widget-body`. That's the reason behind the inclusion of `$scope.toggle = function() { $scope.TextView = !$scope.TextView; };` in the javascript.

I think this is related to `ng-isolate-scope`, which keeps buttons isolated from the body. But I didn't find any other workaround for it.

Reply:
You're right there is actually a bunch of scopes so we usually store such things in `state` scope's property so you can store value `$scope.state.TextView` and it will be accessible in widget header.

Reply:
Awesome. Thank you again. I updated it using `uib-btn-radio`.